app/uk/gov/hmrc/exports/migrations/changelogs/cache/AddDeclarationMetaEntity.scala

```diff
@@ -43,8 +43,7 @@ class AddDeclarationMetaEntity extends MigrationDefinition with Logging {
     MigrationInformation(
       id = s"CEDS-4371 Add DeclarationMeta entity. Move several declaration metadata fields inside it.",
       order = 16,
-      author = "Tom Robinson",
-      runAlways = false
```

Comment:
I think we want to leave these enabled to run until we deploy to the integration environment?